src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +71,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
```

Comment:
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.